### DISCUSSION TITLE
Make tab/window titles start with conscise unique/info such as a keyword/tag  or  media duration, channel & video title

### DISCUSSION THREAD

**Discussion by Gorrrg** [code-for-charity/Wish-Factory#10] [Proposals & Voting, Ranking]
⚬ _PROBLEM_: 

I use Vivaldi because it let's me easily stack Tabs and re-arrange tabs. I often keep many YT videos open as tabs to watch later. I even have an extension that can sort tabs alphabetically, "Tab Sorter". And there's one problem I encounter after a while, the more YT tabs are open, the more complicated it becomes to remember which page title (which is equal to the youtube video title) is related to which channel. So I'd love to be able to sort videos from one channel into one tab stack, without having to check and focus every YouTube tab.

⚬ _SOLUTION_:    

I'm looking for something that will automatically change the tab title / webpage title of YouTube videos pages from just the video title to also have the channel name and/or the release date before the video title. So for instance "Elephant Steals Hat And Eats It" becomes "Daily Dose Of Internet - 2021-02-08 - Elephant Steals Hat And Eats It". I think that should be fairly easy to implement.

 ⚬ _ALTERNATIVES_: 

I don't really think there's an alternative, at least none that I could come up with.

 ⚬ _RELEVANCE / SCOPE_: 

<5% honestly. I guess not a lot of people use tab stacks and Tab Sorter like I do or Vivaldi to begin with. But even if tab sorting and stacking is your goal, I think a lot more people who have a lot of YT tabs open would benefit from this IF they use something like a side panel that displays all the windows and tabs the browser has opened, like Vivaldi offers. The regular tab bar at the top won't show the actual titles anymore since the longer string will be cut off.

⚬ _"SIDE EFFECTS"_:   
 
I think regular Chrome users who don't use any tab management extensions would not want to use this features as well.

⚬ _CONTEXT_:       
Even though I think this is a niche request only a fraction of users will benefit from, I can imagine that the effort to implement it is relatively easy and straight forward, you basically just take two variables that your extension already extracts from the webpage - the channel name and the release date, and add it to a string that is then replacing the tab title. And when of course it should be disabled by default. That's why I'm hoping you are considering this request.

// 
 Thank  you!

SHORT Table | (Summary)     
------------ | -------------   
*Problem*     | Window/tab title are missing the channel name, which makes it hard to sort YT tabs if you have a lot of them open.                                  
*Solution*     | Rewrite the tab title with JS to include the channel name and optionally also the release date
*Alternatives*| ?
*Scope*         | Not a lot of users, basically only those who use Vivaldi or a tab management extension.          
*Side effects*| If the option is enabled the normal tab bar becomes rather useless.
*Context*      | Easy to implement, if it's disabled by default won't hurt anyone, but make some people very happy.

**Comment by Gorrrg** [comment]
Even with normal websites not specifically YouTube but like everything, having a lot of tabs open, the tab titles in the tab bar become really unusable after a while because all you see are the first 6-11 characters. I often use Vivaldi's "Window" sidebar nowadays to change between tabs, because it let's me read the actual website titles again. Managing many tabs is really difficult, there's no perfect solution only things that make life a little bit easier.

**Comment by lucky-ly** [comment]
As "regular Chrome user" (Yandex browser user actually) can say that it would still be useful. I use tooltips on tabs when can't remember, what's there but don't want to click on a tab and start the video and lose "unread" marking on that tab.
Especially if length of the video can be added to the tab title too.
